litr-demo/src/main/java/com/linkedin/android/litr/demo/data/TransformationPresenter.java

```diff
@@ -568,7 +567,7 @@ public void transcodeAudio(@NonNull SourceMedia sourceMedia,
                 AudioTrackFormat trackFormat = (AudioTrackFormat) targetTrack.format;
                 mediaFormat = MediaFormat.createAudioFormat(
                         "audio/mp4a-latm",
-                        trackFormat.samplingRate,
+                        trackFormat.samplingRate / 2,
```

Comment:
I would prefer picking common samplingRates for demo, e.g., 44k -> 8k, or other pairs, instead of a simple half (48K -> 24K?), which might also help find out issue early).

Reply:
The idea here was to guarantee resampling, regardless of source sample rate. I should probably make this configurable in UI.